pennylane/qchem/basis_set.py

```diff
@@ -87,10 +87,21 @@ def atom_basis_data(name, atom):
     >>> print(params)
     [((0, 0, 0), [3.425250914, 0.6239137298, 0.168855404], [0.1543289673, 0.5353281423, 0.4446345422])]
     """
-    basis_sets = {"sto-3g": STO3G, "6-31g": POPLE631G, "STO-3G": STO3G, "6-31G": POPLE631G}
+    basis_sets = {
```

Comment:
This could be made simpler by just using either of the lower/upper case names and just converting the `basis_set` string to either of the cases before doing the comparison using `.lower()` or `.upper()`.

Reply:
Defined the basis sets in `basis_data.py` and added `basis_name = basis_name.lower()` to make sure that whatever the users put will be translated to the lower case format internally.